cookbooks/2d_annulus_visualization/doc/2d_annulus_visualization.md

```diff
@@ -0,0 +1,117 @@
+(sec:cookbooks:2d_annulus_visualization)=
+# Two-dimensional cylindrical annulus visualization
+
+*This section was contributed by Madeleine Kerr.*
+
+This cookbook describes how to use the Python package pyvista to read in time series data from ASPECT .pvd and .pvtu output files (commonly written as 'solution.pvd' or 'solution-00000.pvtu' within the automatically generated [output/solution/] folder). This particular script has the built-in assumption that the model has a 2D cylindrical annulus geometry since it extracts radial and angular velocity from x,y coordinates and velocities and uses the inner and outer radius to threshold the mesh and perform operations.
+
+The input file to generate the model that created the accompanying figures can be found at [cookbooks/twoD_annulus_visualization/2D_annulus_example.prm]. The ASPECT parameter file is a simple modification of the onset_of_convection cookbook contributed by Juliane Dannberg.
```

Comment:
It is very nice that you attribute the individual files to their authors, however we handle authorship citations within the ASPECT project slightly differently. You receive credit for your contribution by adding a changelog entry signed with your name when you contribute a larger piece of code. Potentially you will also be added to the list of authors of the manual, and/or the principal developers of the software. However, all code/prms/manual entries that are contributed to ASPECT become part of the software project as a whole, and are usually not attributed to a single person (except for the line at the top of this file). We chose this convention to make clear that every piece of ASPECT is part of the community code, and everyone has the right to change or improve every piece of ASPECT.

So TLDR: Please do not list the author of every other file in ASPECT that you mention. But please add a changelog entry under your name to `doc/modules/changes`.

```suggestion
The input file to generate the model that created the accompanying figures can be found at [cookbooks/2D_annulus_visualization/2D_annulus_example.prm]. The ASPECT parameter file is a simple modification of the onset_of_convection cookbook.
```